docs/messaging.rst

```diff
@@ -0,0 +1,101 @@
+Messaging (WIP)
+***************
```

Comment:
What is still outstanding? Is there a plan to address that later?

Reply:
Documentation on how to utilizes the `project` and `user` target types is the only thing that was pending at the time; But it doesn't seem like we currently utilize those target types.

Reply:
The initial plan was to mention it for now and improve the documentation once the user and project target types are in use; _I'd like to think that they'd be used for the activity log functionality_